packages/webapp/pages/register.tsx

```diff
@@ -33,18 +33,31 @@ export default function Register(): ReactElement {
     return getFeatureValue(featureFlag, flags);
   };
 
+  const getRedirectUri = (hasFilters?: boolean) => {
+    const uri = router.query.redirect_uri as string;
+
+    if (!uri) {
+      return hasFilters ? '/my-feed' : '/';
+    }
+
+    return hasFilters ? `${uri}?create_filter=true` : uri;
+  };
```

Comment:
I originally thought of adding this variable here: https://github.com/dailydotdev/apps/blob/master/packages/shared/src/components/LoginButtons.tsx#L24
In the login button, rather than messing with the url in registration.
It feels like a more solid implementation, I think, and makes the registration page unaware of everything.
What do you think?

Reply:
Ohh. Very interesting. Let me work on this one first thing!